src/features/Layout/Banners/Demo.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+import { BREAK_POINTS, COLORS, FONT_SIZE, LINE_HEIGHT, SPACING } from '@theme';
+
+export const DemoBanner = styled.div`
+  position: absolute;
+  background: ${COLORS.WARNING_ORANGE};
+  font-weight: bold;
+  font-size: ${FONT_SIZE.MD};
+  padding: ${LINE_HEIGHT.XXS} ${LINE_HEIGHT.BASE} ${LINE_HEIGHT.BASE} ${LINE_HEIGHT.XXS};
```

Comment:
Any reason we are using `LINE_HEIGHT` instead of `SPACING`?

Reply:
Design used sizing that didn't conform to our `SPACING` sizes :shrug: